Modules/Devices/src/Devices.Domain/Entities/Identity.cs

```diff
@@ -6,15 +6,14 @@ namespace Backbone.Modules.Devices.Domain.Entities;
 
 public class Identity
 {
-    public Identity(string? clientId, IdentityAddress address, byte[] publicKey, Tier tier, byte identityVersion)
+    public Identity(string? clientId, IdentityAddress address, byte[] publicKey, byte identityVersion)
```

Comment:
Why did you remove the Tier parameter from the constructor? 
It shouldn't be possible to create an Identity without a Tier.

Reply:
EF needs to create instances of the object and cannot have a navigation property in the constructuor, as far as I could see. The error I got was the following:

```
System.InvalidOperationException: 'No suitable constructor was found for entity type 'Identity'. The following constructors had parameters that could not be bound to properties of the entity type: cannot bind 'tier' in 'Identity(..., Tier tier)'.'
```

Reply:
In that case, we should change it completely (I wanted to suggest that already a few days ago): instead of saving a reference to the Tier, we should only save the TierId. That's the solution DDD suggests anyway (when referencing other Aggregates, only do so via their Id and never the whole object)

Reply:
In this case, will we have both a `Tier` and a `TierId` in the `Identity`?

Reply:
Nope, only the TierId

Reply:
The generated migration does not include a ForeignKey. Should I add it manually, or is there a way to specify that `TierId` must be a valid `Tier.Id`. I looked online but couldn't find anything.